toggle/TemperatureGraph.py

```diff
@@ -108,9 +108,10 @@ def update_temperature_status(self, temp):
       if temp.has_key(tool):
         self.temps[tool]["t_actual"] = temp[tool]["actual"]
         self.temps[tool]["t_target"] = temp[tool]["target"]
-
-    self.lbl_temp.set_text("B:{} T0:{} T1:{}".format(temp["bed"]["actual"], temp["tool0"]["actual"],
-                                                     temp["tool1"]["actual"]))
+    if "tool1" in temp:
```

Comment:
Hmm, I think we can revisit this later, because Revolve will add tool2 in addition to tool1 to the mix.